prime-router/metadata/fhir_mapping/hl7/datatype/HDToExtensionAssigningAuthority.yml

```diff
@@ -0,0 +1,72 @@
+url:
+  type: SYSTEM_URL
+  value: 'assigning-authority'
+
+extension_1-namespaceId:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: namespace-id
+    valueString:
+      type: STRING
+      valueOf: HD.1
+      expressionType: HL7Spec
```

Comment:
Don't have a strong preference, though I personally find it easier to understand with `extension/Extension`.  Logistically it looks like there are a lot fewer uses of it as well.

Reply:
I'd vote for `extension/Extension`, once you're familiar with how it works I agree that it's easier to understand and it saves a few lines per element. Also don't have a strong preference though.

Reply:
Hmmm... so I'm not actually sure we would always be able to use it though?  Do you know how it would work if the extension had a `valueCodeableConcept` where we need to reference a datatype?

Reply:
I don't know how/if we could use it in those cases, it seems to only be set up to work for primitive-valued extensions. Maybe that swings us back to not using `extension/Extension` to avoid having two different ways of doing things?